internal/pkg/template/templates/workloads/partials/cf/vpc-connector.yml

```diff
@@ -0,0 +1,39 @@
+ServiceSecurityGroup:
+  Type: AWS::EC2::SecurityGroup
+  Metadata:
+    'aws:copilot:description': 'A security group for your addons to allow ingress'
+  Properties:
+    GroupDescription: !Join [ '', [ !Ref AppName, '-', !Ref EnvironmentName, !Ref WorkloadName ] ]
+    VpcId:
+      Fn::ImportValue:
+        !Sub '${AppName}-${EnvName}-VpcId'
+    Tags:
+      - Key: Name
+        Value: !Sub 'copilot-${AppName}-${EnvironmentName}-${WorkloadName}'
```

Comment:
Do they allow two SG with the same tag name? I'm thinking of a scenario where the previous stack failed to delete this SG and when you are trying to create the stack again, it could possibly fail if they require such kind of naming uniqueness.

Reply:
For tags it's okay! I just tried this via the console and it's indeed allowed